modules/serverless-create-broker-kn.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * serverless/event_workflows/serverless-using-brokers.adoc
+
+[id="serverless-create-broker-kn_{context}"]
+= Creating a broker using the Knative CLI
+
+.Prerequisites
+* The {ServerlessOperatorName} and Knative Eventing are installed on your {product-title} cluster.
+* You have installed the `kn` CLI.
+
+.Procedure
+* Create the `default` broker:
++
+[source,terminal]
+----
+$ kn broker create <broker_name>
```

Comment:
either use `default` consistently or don't refer to the literal `default` broker